### ISSUE TITLE
[React Native Gallery -> Popup]: Keyboard focus is getting reset on invoking 'Close popup' button.

### ISSUE THREAD

**Issue by YajurG** [microsoft/react-native-gallery#365]
**Repro-Steps:**

Launch the React Native Gallery app.
Navigate to ‘Popup’ button and invoke it.
Navigate to 'Open Popup' button and invoke it.
Observe a popup will be appeared.
Now navigate to 'Close popup' button and invoke it.
Observe keyboard focus.

**Actual Result:**

Keyboard focus is getting reset on invoking 'Close popup' button. After invoking Close popup button, focus is not visible and when user hit tab key focus is landing on the first interactive element i.e., 'Navigation bar' button. 

**Expected Result:**

Keyboard focus should not get reset when user invoke 'Close popup' button. After invoking Close popup button, focus should land on the 'Open popup' button.

**User Impact:**
Users with motor disabilities who rely on keyboard will face difficulties if focus gets reset. 

Internal Bug Link: https://dev.azure.com/microsoft/OS/_workitems/edit/47844034
